File: DriftGame/libs/src/java-universal-tween-engine/tween-engine-api/src/aurelienribon/tweenengine/TweenPaths.java
package aurelienribon.tweenengine;

import aurelienribon.tweenengine.paths.CatmullRom;
import aurelienribon.tweenengine.paths.Linear;

/**
 * Collection of built-in paths.
 */
public interface TweenPaths {
    Linear linear = new Linear();
    CatmullRom catmullRom = new CatmullRom();
}

File: DriftGame/libs/src/java-universal-tween-engine/tween-engine-api/src/aurelienribon/tweenengine/TweenPaths.kt
package aurelienribon.tweenengine

import aurelienribon.tweenengine.paths.CatmullRom
import aurelienribon.tweenengine.paths.Linear

/**
 * Collection of built-in paths.
 */
interface TweenPaths {
    companion object {
        val linear = Linear()
        val catmullRom = CatmullRom()
    }
}
